app/src/main/java/uk/ac/ebi/atlas/experimentpage/tabs/ExperimentPageContentService.java

```diff
@@ -80,7 +88,11 @@ public JsonObject getTsnePlotData(String experimentAccession) {
         result.add("defaultPlotMethodAndParameterisation",
                 GSON.toJsonTree(fetchDefaultPlotMethodAndParameterisation(experimentAccession)));
 
-        result.add("metadata", getMetadata(experimentAccession));
+        var metadata = getMetadata(experimentAccession);
+
+        result.add("metadata", metadata);
```

Comment:
Could you please move and group these lines with the other code where we populate the `result` variable and extract the whole into a private well named method?
There are some related lines below, too.
Thanks